build.gradle

```diff
@@ -1,37 +1,87 @@
+import cuchaz.enigma.command.ConvertMappingsCommand
+
+buildscript {
+	repositories {
+		maven {
+			name "Fabric Repository"
+			url 'https://maven.fabricmc.net'
+		}
+		mavenLocal()
+	}
+
+	dependencies {
+		classpath "cuchaz:enigma:0.15.0"
+	}
+}
+
 plugins {
-    id 'maven'
-    id 'maven-publish'
+	id 'maven'
```

Comment:
What can I do... The indent wasn't consistent...

Reply:
Always use tabs.